sass/util/_space-misc.scss

```diff
@@ -0,0 +1,3 @@
+// additional, ad-hoc spacing utilities added as needed
+
+.lh-32p { line-height: 32px; }
```

Comment:
Would it better to use one of our existing spacing elements for this to keep our classes and the ones from basscss similar?